javascript-custom-methods/calculator.js

```diff
@@ -0,0 +1,27 @@
+/* exported calculator */
+var calculator = {
+  add: function (x, y) {
+    return x + y;
+  },
+  subtract: function (x, y) {
+    return x - y;
+  },
+  multiply: function (x, y) {
+    return x * y;
+  },
+  divide: function (x, y) {
+    return x / y;
+  },
+  square: function (x) {
+    return x * x;
+  },
+  sumAll: function (numbers) {
+    var sum = numbers[0];
+    for (var i = 1; i < numbers.length; i++) {
+      sum += numbers[i];
+    } return sum;
```

Comment:
I recommend moving your `return` statement to the line below. It's really hard to read when it's like this, most developers will just skim through the code looking at each new line, and some will probably miss this return statement if it's formatted like this